deezerapp/src/app/models/tracklist.ts

```diff
@@ -0,0 +1,5 @@
+import { Track } from "./track";
+
+export class Tracklist {
```

Comment:
Because the only member this class has is an array of `tracks` we can simplify by removing it and just using `tracks: Track[]` instead.